e2e-tests/portal/cypress/e2e/journeys/checker/cover-end-date/deal.js

```diff
@@ -1,7 +1,8 @@
-const mockDeal = require('../../../../fixtures/deal');
-const CONSTANTS = require('../../../../fixtures/constants');
-const FIXTURE_CONSTANTS = require('../../../../../../e2e-fixtures/constants.fixture');
-const { today, tomorrow } = require('../../../../../../e2e-fixtures/dateConstants');
+import { BOND_FACILITY_TYPE } from '@ukef/dtfs2-common';
+import mockDeal from '../../../../fixtures/deal';
```

Comment:
Seems that we have a mix and match throughout E2E tests of using `import`, `export default`, `export const` and `require` and `module.exports`.

I assume we want to continue moving towards ES6 imports rather than requires.

Whilst I was updating these files I changed them to use imports.